install/linux/docker-ee/suse.md

```diff
@@ -164,11 +164,14 @@ Before you install Docker EE for the first time on a new host machine, you need
 to set up the Docker repository. Afterward, you can install and update Docker EE
 from the repository.
 
-There are currently two versions of Docker EE Engine available:
+The current version of Docker EE Engine is 18.09 for using with Docker Enterprise Edition 2.1.
 
-* 18.03 - Use this version if you're only running Docker EE Engine.
-* 17.06 - Use this version if you're using Docker Enterprise Edition 2.0 (Docker
-Engine, UCP, and DTR).
+> ***NOTE:*** There are two older versions of Docker EE Engine available:
+> * 18.03 - Use this version if you're only running Docker EE Engine.
+> * 17.06 - Use this version if you're using Docker Enterprise Edition 2.0
```

Comment:
And here (although already less confusing); wondering if we should emphasize that it's only needed if they want to stick to Docker Enterprise 2.0. (And for Docker Enterprise Edition 2.1, they should install/upgrade to 18.09)

Reply:
Changed this on all the docs. Can you please let me know how the wording looks @thaJeztah?

Reply:
@thaJeztah I've also added links back to the older documentation.